apps/ssr-tests-v9/src/test.ts

```diff
@@ -15,18 +15,18 @@ async function test(): Promise<void> {
   const startTime = process.hrtime();
   console.log('Starting a browser...');
 
+  const htmlPath = path.resolve(__dirname, '..', 'dist', 'index.html');
+
+  if (!fs.existsSync(htmlPath)) {
+    throw new Error('"dist/index.html" does not exist, please run "yarn build" first');
+  }
+
```

Comment:
launching browser is not necessary if pre-requirement are not met. exiting early